src/demo/index.ts

```diff
@@ -102,15 +97,24 @@ export function startBpmnVisualization(config: BpmnVisualizationDemoConfiguratio
 
   const parameters = new URLSearchParams(window.location.search);
 
-  fitOnLoad = parameters.get('fitOnLoad') == 'true';
-  log(`Configure 'fit on load' to ${fitOnLoad}`);
+  log('Configure load options');
+  const parameterFitType = parameters.get('fitType');
+  if (parameterFitType) {
+    options.fitType = FitType[parameterFitType as keyof typeof FitType];
+  } else {
+    const selectorFitType = document.getElementById('fitType') as HTMLSelectElement;
+    if (selectorFitType) {
+      options.fitType = FitType[selectorFitType.options[selectorFitType.selectedIndex].value as keyof typeof FitType];
```

Comment:
:warning: this only works in the index.html page so this code should be moved to index.js.
Otherwise, errors occur in other html pages.